app/models/miq_event.rb

```diff
@@ -30,8 +30,10 @@ def self.raise_evm_event(target, raw_event, inputs = {}, options = {})
     raise "Unable to find object for target: [#{target}]" unless target
 
     event = normalize_event(raw_event.to_s)
-    # TODO: how to handle unknow MiqEvent - event not defined in MiqEventDefinition?
-    return if event == 'unknown'
+    if event == 'unknown' && target.class.base_class.in?(SUPPORTED_POLICY_AND_ALERT_CLASSES)
+      _log.warn("Event #{raw_event} for class [#{target.class.name}] id [#{target.id}] is not raised: #{raw_event} is not defined in MiqEventDefinition")
```

Comment:
Suggest `_log.warn("Event #{raw_event} for class [#{target.class.name}] id [#{target.id}] was not raised: #{raw_event} is not defined in MiqEventDefinition")`